tests/controllers/S3LinkController.ts

```diff
@@ -0,0 +1,37 @@
+import {describe, test} from '@jest/globals';
+import {containerBuilder} from '../testContainer';
+import {Request, Response} from 'express';
+import fn = jest.fn;
+import S3LinkController from '../../src/controllers/S3LinkController';
+
+describe('S3 Link Controller', () => {
+  const container = containerBuilder();
+  const s3LinkController = container.get('s3LinkController') as S3LinkController;
+
+  test('run action Default', () => {
+    const request = {
+      body: {
+        frames: ['image22.jpg', 'image33.jpg']
+      }
+    } as Request;
+    const response = {
+      json: fn((data) => {
+        // const fileNames = Object.keys(data);
+
+        expect(data).toHaveLength(3);
```

Comment:
everything looks good, I don't understand why data.length sould be 3 if we are passing 2 pictures
data right now should be ['image22link','image33link'] right?

Reply:
because this is the faces not the images :)